pep-0480.txt

```diff
@@ -859,7 +874,7 @@ References
 .. [1] https://www.python.org/dev/peps/pep-0458/
 .. [2] https://isis.poly.edu/~jcappos/papers/samuel_tuf_ccs_2010.pdf
```

Comment:
This link and more could be very outdated. Could we please update them and use the same URLs we did for 458?